lib/ansible/modules/cloud/spotinst/spotinst_aws_elastigroup.py

```diff
@@ -9,7 +9,7 @@
 DOCUMENTATION = """
 ---
 module: spotinst_aws_elastigroup
-version_added: 2.5
+version_added: 2.8
```

Comment:
The version_added on an existing module should not be changed.

Reply:
ok just changed it back, sorry about that!!